draft-ietf-quic-transport.md

```diff
@@ -1569,12 +1569,15 @@ An endpoint MUST NOT initiate connection migration before the handshake is
 finished and the endpoint has 1-RTT keys.
 
 This document limits migration of connections to new client addresses.
-Migrating a connection to a new server address is left for future work. If a
-client receives packets from an unknown server address, the client MAY discard
-these packets.
+Additionally, clients are responsible for initiating all migrations.  Servers
+do not send non-probing packets (see {{probing}}) toward a client address until
+it sees a non-probing packet from that address.  Additionally, if a client
```

Comment:
Two "additionally"s in three lines seems a bit redundant, as well as repetitive.  Perhaps drop the second one?